src/index.html

```diff
@@ -11,6 +11,34 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      data-qa="big"
+      class="big-form search-form"
+    >
+      <label for="search" class="search-wrapper">
+        <input
+          class="search-input search-input__big"
+          type="text"
+          data-qa="keypress"
+          placeholder="Try &quot;Los Angeles&quot;"
+        >
+      </label>
+    </form>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      data-qa="small"
+      class="small-form search-form"
+    >
```

Comment:
Use class names according to checklist
![image](https://user-images.githubusercontent.com/55200530/227767343-6d71d6b6-16f5-4bc9-af3c-58b0e4860ae7.png)